mmv1/third_party/terraform/services/compute/data_source_google_compute_subnetwork.go.tmpl

```diff
@@ -100,6 +104,10 @@ func dataSourceGoogleComputeSubnetworkRead(d *schema.ResourceData, meta interfac
 		return transport_tpg.HandleDataSourceNotFoundError(err, d, fmt.Sprintf("Subnetwork Not Found : %s", name), id)
 	}
 
+	if err := d.Set("numeric_id", subnetwork.Id); err != nil {
```

Comment:
I didn't seem to need to convert from `int64` to `int` here, even though I did when I was doing string conversion. Let me know if there should be an `int(subnetwork.Id)` here.